integration-tests/src/test/java/tachyon/client/LocalBlockInStreamIntegrationTest.java

```diff
@@ -49,12 +49,12 @@
 
   @ClassRule
   public static LocalTachyonClusterResource sLocalTachyonClusterResource =
-      new LocalTachyonClusterResource(Constants.MB, Constants.KB, Constants.MB);
-  private static TachyonFileSystem sTfs = null;
-  private static OutStreamOptions sWriteBoth;
-  private static OutStreamOptions sWriteTachyon;
-  private static InStreamOptions sReadNoCache;
-  private static InStreamOptions sReadCache;
+      new LocalTachyonClusterResource(Constants.GB, Constants.KB, Constants.GB);
```

Comment:
I was keeping consistency with `BufferedBlockInStream`, but testing larger block size probably doesn't make a difference. I'll revert this change and we can reconcile the sizes later.